src/components/ServiceDetails/index.js

```diff
@@ -74,14 +75,20 @@ class ServiceDetails extends Component {
   }
 }
 
-const mapStateToProps = state => ({
-  service: state.serviceDetailsReducer,
-  pricing: pricing(state),
-});
+const mapStateToProps = (state, ownProps) => {
+  const {
+    match: {
+      params: { orgId, serviceId },
+    },
+  } = ownProps;
+
+  return { service: serviceDetails(state, orgId, serviceId), pricing: pricing(state) };
+};
 
 const mapDispatchToProps = dispatch => ({
   fetchServiceDetails: args => dispatch(serviceDetailsActions.fetchServiceDetails({ ...args })),
   resetServiceDetails: () => dispatch(serviceDetailsActions.resetServiceDetails),
+  fetchMeteringData: args => dispatch(serviceDetailsActions.fetchMeteringData({ ...args })),
```

Comment:
Do you need `resetServiceDetails` and `fetchMeteringData` anymore here?

Reply:
removed https://github.com/singnet/refactored-dapp/pull/160/commits/1c6427e6799311714b153c05dccc87e481d1e1af